Running in wrapper mode for '314'

Running python job.

[name: "CPU-QUOTA"
int_val: 53
int_val: 28
int_val: 60
int_val: 25
int_val: 92
int_val: 68
int_val: 63
, name: "DISK"
int_val: 33
int_val: 7
int_val: 12
int_val: 8
int_val: 26
int_val: 11
int_val: 22
, name: "MEMORY"
int_val: 7
int_val: 28
int_val: 9
int_val: 13
int_val: 35
int_val: 38
int_val: 36
, name: "NET"
int_val: 8
int_val: 10
int_val: 6
int_val: 14
int_val: 9
int_val: 33
int_val: 38
]
The paramters are {u'CPU-QUOTA': array([53, 28, 60, 25, 92, 68, 63]), u'NET': array([ 8, 10,  6, 14,  9, 33, 38]), u'DISK': array([33,  7, 12,  8, 26, 11, 22]), u'MEMORY': array([ 7, 28,  9, 13, 35, 38, 36])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
Setting cpu quota to 70000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, NET 51200.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, DISK 5872025.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER d824175f3227, MEMORY 1050000000.0
Recovering the memory utilization
Setting cpu quota to 132500 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, NET 40960.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, DISK 27682406.4
STRESSING VM_IP 13.56.14.158 AND CONTAINER cd40be619604, MEMORY 262500000.0
Recovering the memory utilization
Setting cpu quota to 150000 out of 250000
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, NET 30720.0
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, DISK 10066329.6
STRESSING VM_IP 13.56.14.158 AND CONTAINER c21ccad5695a, MEMORY 337500000.0
Recovering the memory utilization
Setting cpu quota to 132500 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, NET 40960.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, DISK 27682406.4
STRESSING VM_IP 18.144.43.31 AND CONTAINER 01a0240fb06e, MEMORY 262500000.0
Recovering the memory utilization
Setting cpu quota to 150000 out of 250000
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, NET 30720.0
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, DISK 10066329.6
STRESSING VM_IP 18.144.43.31 AND CONTAINER d4df0b3c1712, MEMORY 337500000.0
Recovering the memory utilization
Using 5 trials
INVALID EXPERIMENT TYPE: todo-app

----------------------------------------
Problem running the job:
(<type 'exceptions.SystemExit'>, SystemExit(None,), <traceback object at 0x7effc46ec638>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 204, in explore_spearmint
    experiment_results = run_experiment.measure_runtime(workload_config, experiment_trials)
  File "/home/ubuntu/throttlebot/src/run_experiment.py", line 37, in measure_runtime
    exit()
  File "/usr/lib/python2.7/site.py", line 375, in __call__
    raise SystemExit(code)
SystemExit: None
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 15.19 seconds.
